vendor/assets/javascripts/local-time.js

```diff
@@ -0,0 +1,108 @@
+// modifications marked with "login.gov" original here: https://github.com/basecamp/local_time/blob/master/app/assets/javascripts/local-time.js
+
```

Comment:
Are you sure you want to put this in vendor/assets/javascripts?  It's not a verbatim vendor file.  We made modifications.

Reply:
Yeah, it's tricky, but it looks like it's primarily 3rd party code and a lot of it has been minified. Unless we want to track down the source, clean it up to make it match our standards for JS, and then move that in here, I think it is best to vendor it.